src/bioregistry/data/bioregistry.json

```diff
@@ -70498,6 +70498,56 @@
       }
     ]
   },
+  "mirgenedb": {
+    "contact": {
+      "email": "BastianFromm@gmail.com",
+      "name": "Bastian Fromm",
+      "orcid": "0000-0003-0352-3037"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "mircoRNA Gene Database is a collection of animal microRNA genes. This database allows researchers to analyze mircoRNA properties like secondary structure and features in a phylogenetic context.",
+    "example": "hsa/Let-7-P1b",
+    "github_request_issue": 1393,
+    "homepage": "https://mirgenedb.org/",
+    "name": "microRNA Gene Database",
+    "pattern": "^[a-zA-Z0-9-]+-[a-zA-Z0-9-]+$",
```

Comment:
This pattern doesn't entirely make sense because the `-` is matched as a character in the first `[a-zA-Z0-9-]` block as many times as needed, so the whole second part of the pattern `-[a-zA-Z0-9-]+` is redundant. It also doesn't exclude an identifier starting with a `-`, etc. It should be something like `^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)+$`